### DISCUSSION TITLE
Search option for Docs

### DISCUSSION THREAD

**Discussion by jensolafkoch** [livewire/flux#211] [Feature Requests] [answered]
Hi,

I'm so used to CTRL+K, I always wonder that nothing happens on the Flux docs page. 

I'd like to see a search option on https://fluxui.dev/docs sometime down the road. (And maybe a video explaining the implementation.) :-)

**Comment by aurawindsurfing** [comment]
Search is much needed please

**Comment by calebporzio** [comment, marked as answer]
This is done -  thanks!

**Reply by jeffchown** [reply]
Nice!  Thanks @calebporzio 🥳
